client/src/components/App.js

```diff
@@ -12,24 +16,24 @@ const App = (props) => {
   const [currentUser, setCurrentUser] = useState(undefined);
   const fetchCurrentUser = async () => {
     try {
-      const user = await getCurrentUser()
-      setCurrentUser(user)
-    } catch(err) {
-      setCurrentUser(null)
+      const user = await getCurrentUser();
+      setCurrentUser(user);
+    } catch (err) {
+      setCurrentUser(null);
     }
-  }
+  };
 
   useEffect(() => {
-    fetchCurrentUser()
-  }, [])
+    fetchCurrentUser();
+  }, []);
 
   return (
     <Router>
       <TopBar user={currentUser} />
       <Switch>
-        <Route exact path="/">
-          <h2>Hello from react</h2>
-        </Route>
+        <Route exact path="/" component={HomePage} />
+        <Route exact path="/categories" component={CategoriesList} />
```

Comment:
What's the difference between the Home Page and Category List? It looks like the Home Page just displays the list, but we also have a route to see the list.

If we're adding a home page to this app, that would be a separate feature from this. This branch is already starting to get out of scope, that it involves categories index, show pages, pasta, etc. We should break each of these down into smaller, bite sized feature branches

Reply:
Thanks for the comment. It was king of an on the fly addition because our Routes on the app page started to feel a little messy, but we will keep this in mind for the future.